mteb/tasks/PairClassification/fra/gqnli.py

```diff
@@ -0,0 +1,72 @@
+from datasets import load_dataset
+from mteb.abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class GqnliTask(AbsTaskPairClassification):
+    metadata = TaskMetadata(
+        name="gqnli",
```

Comment:
```suggestion
        name="GqnliPairClassification",
```
and rename file to match